docs/source/installation/win.rst

```diff
@@ -5,6 +5,18 @@ First, you need to install Chocolatey, which is a package manager for Windows
 systems.  Please refer to `this link <https://chocolatey.org/install>`_ for
 instructions.
 
+Installing using Chocolatey
+***************************
+
+You can install chocolatey very easily using chocolatey. Typing the command install Manim.
```

Comment:
```suggestion
You can install manim very easily using chocolatey, by typing the following command.
```

Reply:
This suggestion was not applied?

Reply:
I think it should be applied. I was drowsy when I was typing that and so it was wrong.

Reply:
Lol I left it and then merged this. :joy:  Yes we should make this change.

Reply:
Who _is _ making an PR for this?